src/sql/sqlops.proposed.d.ts

```diff
@@ -315,7 +315,7 @@ declare module 'sqlops' {
 		actions?: ActionDescriptor[];
 		status?: StatusIndicator;
 		selected?: boolean;
-		cardType: CardType;
+		cardType?: CardType;
```

Comment:
It might be good to add comments to this and the other properties here about what their defaults are since I know people writing extensions had asked for more useful doc comments before